docs/8-veba-troubleshooting.md

```diff
@@ -0,0 +1,277 @@
+# vCenter Event Broker Appliance - Troubleshooting
+
+## Table of Contents
+[Requirements](#requirements)<br>
+[Troubleshooting an initial deployment](#troubleshooting-an-initial-deployment)<br>
+[OpenFaaS Function troubleshooting](#openfaas-function-troubleshooting)
+
+## Requirements
+
+You must log on to the vCenter Event Broker appliance as root. You can do this from the console. If you want SSH access, execute the following command:
+
+```
+systemctl start sshd 
+```
+
+This turns on the SSH daemon but does not enable it to start on appliance boot. You should now be able to SSH into the appliance. 
+
+If you wish to disable the SSH daemon when you are done troubleshooting, execute the following command:
+
+```
+systemctl stop sshd 
+```
+<BR>
+
+## Troubleshooting an initial deployment
+
+If the appliance is not working immediately after deployment, the first thing to do is check your Kubernetes pods. 
+
+```
+kubectl get pods -A
+```
+
+Here is the command output:
+
+```
+NAMESPACE        NAME                                   READY   STATUS          RESTARTS        AGE
+kube-system      coredns-584795fc57-hcvxh               1/1     Running              1          4d15h
+kube-system      coredns-584795fc57-hf72w               1/1     Running              1          4d15h
+kube-system      etcd-veba02                            1/1     Running              1          4d15h
+kube-system      kube-apiserver-veba02                  1/1     Running              1          4d15h
+kube-system      kube-controller-manager-veba02         1/1     Running              1          4d15h
+kube-system      kube-proxy-fj47p                       1/1     Running              1          4d15h
+kube-system      kube-scheduler-veba02                  1/1     Running              1          4d15h
+kube-system      weave-net-vs8ls                        2/2     Running              4          4d15h
+projectcontour   contour-5cddfc8f6-8hzd6                1/1     Running              1          4d15h
+projectcontour   contour-5cddfc8f6-jq7d8                1/1     Running              1          4d15h
+projectcontour   contour-certgen-f92l5                  0/1     Completed            0          4d15h
+projectcontour   envoy-gcmqt                            1/1     Running              1          4d15h
+vmware           tinywww-7fcfc6fb94-mfltm               1/1     Running              1          4d15h
+vmware           vmware-event-router-5dd9c8f858-5c9mh   0/1     CrashLoopBackoff     6          4d13h
+```
+<B>Note:</B> The status ```Completed``` of the container ```contour-certgen-f92l5``` is expected after successful appliance deployment.
```

Comment:
I don't know whether we have established a best practice for notes. I usually use quoted text so it stands out more.

```
> **Note:** Text
```

Reply:
Fixed